ros2action/ros2action/verb/echo.py

```diff
@@ -187,26 +195,26 @@ def message_handler():
             # specifying QoS of subscription to replace the current fixed QoS.
             if ActionInterfaces.GOAL_SERVICE.value in output_interface_list:
                 send_goal_event_sub: Subscription[EventMessage] = node.create_subscription(
-                        send_goal_event_msg_type,
-                        send_goal_event_topic,
-                        self._send_goal_subscriber_callback,
-                        qos_profile_services_default)
+                    send_goal_event_msg_type,
+                    send_goal_event_topic,
+                    self._send_goal_subscriber_callback,
+                    qos_profile)
```

Comment:
Note that QoS configuration from command line arguments are only applied to `services`, since status and feedback topics have integrated QoS configuration that user cannot change. if we apply the user specified QoS configuration for the topics status and feedback, it can fail to create the subscription because of the QoS incompatibility.